src/macaron/malware_analyzer/pypi_heuristics/metadata/wheel_absence.py

```diff
@@ -64,21 +67,45 @@ def analyze(self, pypi_package_json: PyPIPackageJsonAsset) -> tuple[HeuristicRes
                 logger.debug(error_msg)
                 raise HeuristicAnalyzerValueError(error_msg)
 
-        release_files: list[JsonType] = []
+        file_server_links: list[JsonType] = []
+        inspector_links: list[JsonType] = []
         wheel_present: bool = False
 
         try:
             for release_metadata in releases[version]:
                 if release_metadata["packagetype"] == self.WHEEL:
                     wheel_present = True
 
-                release_files.append(release_metadata["filename"])
+                name = json_extract(pypi_package_json.package_json, ["info", "name"], str)
+                if name is None:
+                    error_msg = "There is no 'name' field for this package."
+                    logger.debug(error_msg)
+                    raise HeuristicAnalyzerValueError(error_msg)
+
+                # include the pypi inspector link, which uses the same suffix of
+                # packages/{blake2b_256}/file_name
+                inspector_prefix = f"{self.INSPECTOR_PREFIX}{name.lower()}/{version}/"
+                inspector_link = release_metadata["url"].replace(self.PYPI_PREFIX, inspector_prefix)
+
+                # use a head request because we don't care about the response contents
+                if send_head_http_raw(inspector_link) is None:
+                    inspector_link = None
+
+                file_server_links.append(release_metadata["url"])
+                inspector_links.append(inspector_link)
+
         except KeyError as error:
             error_msg = f"The version {version} is not available as a release."
             logger.debug(error_msg)
             raise HeuristicAnalyzerValueError(error_msg) from error
 
+        detail_info: dict[str, JsonType] = {
+            "version": version,
+            "file_server_links": file_server_links,
```

Comment:
I checked the results of the other heuristics. We already store the version and `file_server_links` as part of the JSON metadata returned by PyPI, so we can just keep `inspector_links`.